GooglePlayInstant/Editor/QuickDeploy/LoadingBarGenerator.cs

```diff
@@ -69,12 +73,27 @@ public static LoadingBar GenerateLoadingBar()
             loadingBar.ProgressHolder.transform.SetParent(loadingBar.transform, false);
             SetAnchorsToScaleWithParent(loadingBar.ProgressHolder);
 
+            var scrollingFillRectTransform = GenerateScrollingFill();
+            scrollingFillRectTransform.transform.SetParent(loadingBar.ProgressHolder, false);
+            scrollingFillRectTransform.SetAsFirstSibling(); // The scrolling fill should be behind the progress fill.
+
             var loadingBarRectTransform = loadingBarObject.GetComponent<RectTransform>();
             SetAnchorsToScaleWithParent(loadingBarRectTransform);
 
             return loadingBar;
         }
 
+        private static RectTransform GenerateScrollingFill()
+        {
+            var scrollingFillObject = GenerateUiObject(ScrollingFillName);
+            var scrollingFillImage = scrollingFillObject.AddComponent<RawImage>();
+            scrollingFillImage.texture = FindLoadingTileTexture();

```

Comment:
If this returns null should we abort adding the scrolling fill? Or should that case be an exception that crashes the app?

Reply:
As discussed offline, we will throw an exception if we cannot find either the loading tile or the replay button assets.